lib/Functional/Zip.ark

```diff
@@ -0,0 +1,9 @@
+(let zip (fun (a b) {
+    (let m (min (len a) (len b) ))

```

Comment:
since `zip` needs `min`, it should import the corresponding file from Math/, because each file should be able to work on its own

Reply:
Good catch. I've added it in. I thought that since the tests passed, I didn't need to import anything, but I think that since the functional tests are last in line in the list of tests, the `min` function could've been loaded from before.